pyiron/lammps/control.py

```diff
@@ -415,6 +415,7 @@ def calc_md(
         tloop=None,
         initial_temperature=None,
         langevin=False,
+        drag=None,
```

Comment:
Can we move these changes to a separate pull request? they have nothing todo with the structure creator.

Reply:
Ack! Of course. This was just a git error from being on the wrong branch at some point.